modules/solace/src/main/java/org/testcontainers/containers/Service.java

```diff
@@ -0,0 +1,69 @@
+package org.testcontainers.containers;
+
+/**
+ * Services that are supported by Testcontainers implementation
+ *
+ * @author Tomasz Forys
+ */
+public enum Service {
+    /**
+     AMQP service
+     */
+    AMQP("amqp", 5672, "amqp", false),
+    /**
+     MQTT service
+     */
+    MQTT("mqtt", 1883, "tcp", false),
+    /**
+     REST service
+     */
+    REST("rest", 9000, "http", false),
+    /**
+     SMF service
+     */
+    SMF("smf", 55555, "tcp", true),
+    /**
+     SMF with SSL
+     */
+    SMF_SSL("smf", 55443, "tcps", true);
```

Comment:
The name are explicits already.

```suggestion
    AMQP("amqp", 5672, "amqp", false),
    MQTT("mqtt", 1883, "tcp", false),
    REST("rest", 9000, "http", false),
    SMF("smf", 55555, "tcp", true),
    SMF_SSL("smf", 55443, "tcps", true);
```